checklist_plus/text_generation/llm.py

```
@@ -0,0 +1,346 @@
+
+class LLMTextGenerator(TextGenerator):
+    """
+    LLM-based TextGenerator that implements the same interface as TextGenerator
+    but uses LLM for mask filling instead of masked language models.
+    """
+
+    def __init__(self,
+                 llm_client: LLM | None = None,
+                 openai_api_key: str | None = None,
+                 model_name: str = "gpt-3.5-turbo",
+                 **kwargs):
```

Ruff (0.12.2)

Unused method argument: `kwargs`

(ARG002)